src/structures/member.ts

```diff
@@ -60,39 +71,32 @@ const baseMember: Partial<Member> = {
 };
 
 export async function createMemberStruct(
-  data: MemberCreatePayload,
+  data: Omit<DiscordGuildMember, "user"> & { user: DiscordUser },
```

Comment:
Why is this omitted and added back?

Reply:
because user is optional in DiscordGuildMember

Reply:
You should add a comment explaining that, why it is optional non-optional for createMemberStruct, maybe a note from Discord docs.

Reply:
https://github.com/discordeno/discordeno/pull/739/files/fb9ace6d11d790800a87a2280c99f25e732ec83d#diff-f3eedaf51b97c44f868e9f10f7e2f15d0948c19a362d90ab252fe00415074750